app/views/issues/_form.html.erb

```diff
@@ -32,6 +32,9 @@
   <%= render 'tag_input', f: f %>
 
   <div class="form-actions">
+    <% if params[:redirect_to] %>
+      <%= hidden_field :issue, :redirect_to, value: params[:redirect_to] %>
```

Comment:
```suggestion
      <%= hidden_field :redirect_to, value: params[:redirect_to] %>
```

No need to nest this under `:issue`. Note that we're passing all values under `:issue` as attributes for the issue record.

To illustrate, if nested under issue, the params will look like:
```
{
  issue:  {
    text: '...',
    redirect_to: 'somewhere'
  }
}
```
the controller will try to update the issue's `redirect_to` method. It needs to be:
```
{
  issue:  {
    text: '...'
  },
  redirect_to: 'somewhere'
}
```

Reply:
got it, made the change